tests/test_postgres.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+set -xeo pipefail
+
+source tests/common.sh
+
+function finish {
+  echo "Cleaning up postgres"
+  kubectl delete -f tests/test_crs/valid_postgres.yaml
+  kubectl delete csv --all -n ripsaw
+  #kubectl delete secret 1979710-benchmark-operator-ci-pull-secret -n ripsaw
+  marketplace_cleanup
+  delete_operator
+}
+
+# Two arguments are 'pod label' and 'timeout in seconds'
+function get_pod () {
+  counter=0
+  sleep_time=5
+  counter_max=$(( $2 / sleep_time ))
+  pod_name="False"
+  until [ $pod_name != "False" ] ; do
+    sleep $sleep_time
+    pod_name=$(kubectl get pods -l $1 --namespace ripsaw -o name | cut -d/ -f2)
+    if [ -z $pod_name ]; then
+      pod_name="False"
+    fi
+    counter=$(( counter+1 ))
+    if [ $counter -eq $counter_max ]; then
+      return 1
+    fi
+  done
+  echo $pod_name
+  return 0
+}
+
+trap finish EXIT
+
+
+# Note we don't test persistent storage here
+function functional_test_postgres{
+  apply_operator
```

Comment:
CI chokes on this line http://porthos.perf.lab.eng.rdu2.redhat.com:8080/job/machinist_githubpr_minikube/246/console